### PR TITLE
Upgrade version of mssql to fix a bug of date with mssql driver (tedious)

### PR DESCRIPTION
**Pull request by v1d3rm3**
### Description of change

Hello! I'm sending a PR to solve a problem in Date, as referenced here https://github.com/tediousjs/tedious/issues/1249.  Unfortanalety, it'll not be a version to solve the problem in `mssql v6`, so i'm sending this upgrade for `next´ branch, because i'm using this version in my job a long time motivated by this issue (https://github.com/typeorm/typeorm/issues/4270).

In PR, i did the following things (sorry for not separate them). First i upgraded mssql version to v7. Then i correct the way to send name parameters to mssql (tedious aparently not accept name as numbers, only strings).

### Pull-Request Checklist

- [x] (IT FOR NEXT BRANCH) Code is up-to-date with the `master` branch
- [x] `npm run lint` passes with this change
- [x] `npm run test` passes with this change (partially)
- [x] This pull request links relevant issues as `Fixes #0000`
- [x] There are new or updated unit tests validating the change
- [x] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by v1d3rm3** [comment]
@AlexMesser , @pleerock. Some help here, please?

**Comment by AlexMesser** [comment]
Thank you for contribution.

Since it's not clear when `@next` is going to be merged into `master` right now, it's better if you do this PR against master. If its a breaking change we need to add additional code to make it backward-compatible. Also please remove all formatting from the PR to make it easier to review this PR. Thank you.

**Comment by v1d3rm3** [comment]
In my job, we're using the `next` branch, and we're awaiting for these changes. We only follow what collaborators said before https://github.com/typeorm/typeorm/issues/4270#issuecomment-501207213. We cannot use master code cause of error that exists until now. I think 

Changes are identified by commits: https://github.com/typeorm/typeorm/pull/7600/commits/98671c895a6070a137937411cae26372a1ed3b6c and https://github.com/typeorm/typeorm/pull/7600/commits/f2e535cf44f568c555401e54e74aa96e8edd863a.

I decide to fix linting as format, because it's present by pr checklist.

Can you merge this in `next`? After that, i can make cherry picks to do changes in master.

**Review by imnotjames — CHANGES_REQUESTED** [review]
Remove the formatting changes.

File: src/util/DateUtils.ts [hunk 1]
```diff
@@ -33,7 +34,21 @@ export class DateUtils {
      * Converts given value into date object.
      */
     static mixedDateToDate(mixedDate: Date|string, toUtc: boolean = false, useMilliseconds = true): Date {
-        let date = typeof mixedDate === "string" ? new Date(mixedDate) : mixedDate;
+        /**
```
Comment (imnotjames) [inline src/util/DateUtils.ts:37]: If you want to include this change, it's better to do so in a separate r

**Pull request closed without merging**